examples/pokedex/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# production
+/build
```

Comment:
Remove `/build` form gitignore, build the example once (npm run build), and commit again. The committed build folder is required by GitHub pages for the live example.